test/utils.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2024 R. Kent James <kent@caspia.com>
```

Comment:
This file looks like the content is primarily moved content and not newly written so it should retain the original copyright owner.

Reply:
OK